### ISSUE TITLE
calicoctl profile rules example is invalid

### ISSUE THREAD

**Issue by micahhausler** [projectcalico/calico-containers#1196]
The help message for profile rule  manipulation outputs an invalid ruleset

```
  $ calicoctl profile add only-local-pings
  $ calicoctl profile only-local-pings rule add inbound deny icmp
  $ calicoctl profile only-local-pings rule add inbound --at=0 allow from 192.168.0.0/16
```

The last command is invalid and only produces the help message 

```
$ calicoctl version
0.22.0
```

**Comment by fasaxc** [comment]
Should be `... from cidr 192.168.0.0/16` IIRC

**Comment by heschlie** [comment]
Fixed in #1234
